docs/app/partials/layout-container.tmpl.html

```diff
@@ -1,28 +1,11 @@
 <div ng-controller="LayoutCtrl" class="layout-content" ng-cloak>
 
-  <h3>Overview</h3>
-  <p>
-    Angular Material's responsive CSS layout is built on
-    <a href="http://www.w3.org/TR/css3-flexbox/">flexbox</a>.
-  </p>
+  <h3>Layout and Containers</h3>
 
   <p>
-    The layout system is based upon element attributes rather than CSS classes.
-    Attributes provide an easy way to set a value (eg. <code>layout="row"</code>) and
-    help us separate concerns: attributes define layout, and classes define styling.
-  </p>
-  <p>
-    <i>
-      Note: Due to performance problems related to attribute selectors in IE11, our attributes are converted
-      to class selectors at runtime. You should continue to use layout attribute selectors in your code.
-    </i>
-  </p>
-
-  <h3>Layout Attribute</h3>
-  <p>
-    Use the <code>layout</code> attribute on an element to arrange its children
+    Use the <code>layout</code> attribute on an container element to arrange specify the layout direction for its children:
     horizontally in a row (<code>layout="row"</code>) or vertically in a column (<code>layout="column"</code>).
-    Row layout is the default if you specify the <code>layout</code> attribute without a value.
+    Note that <code>row</code> the default layout direction if you specify the <code>layout</code> attribute without a value.
```

Comment:
Typo, add `is`: "Note that `<code>row</code>` **is** the default layout direction if..."